rasgotransforms/rasgotransforms/render.py

```diff
@@ -0,0 +1,93 @@
+"""
+Functions to render Transform Templates
+"""
+from typing import Dict, Any, Optional
+from pathlib import Path
+from .main import DataWarehouse
+from .dtypes import DTYPES
+from importlib import machinery, util
+import os
+import re
+
+
+def get_dtype(dtype: str) -> str:
+    return DTYPES[dtype.lower()]
+
+
+def get_dw_type(dw_type: str) -> Optional[DataWarehouse]:
+    if dw_type:
+        try:
+            return DataWarehouse[dw_type.upper()]
+        except KeyError:
+            raise Exception(f'Unsupported DataWarehouse type: {dw_type}')
+    else:
+        return None
+
+
+def get_path(transform_name: str, dw_type: Optional[DataWarehouse] = None) -> Optional[str]:
+    root_dir = os.path.dirname(__file__)
+    if dw_type:
+        function_path = Path(root_dir, 'transforms', transform_name, dw_type.value,
+                             f'{transform_name}.py')
+        if os.path.exists(function_path):
+            return str(function_path.absolute())
+    function_path = Path(root_dir, 'transforms', transform_name, f'{transform_name}.py')
+    if not os.path.exists(function_path):
+        return None
+    return str(function_path.absolute())
```

Comment:
```suggestion
def get_path(transform_name: str, dw_type: Optional[DataWarehouse] = None) -> Optional[str]:
    transform_dir = Path(__file__).parent / "transforms" / transform_name
    function_path = transform_dir / dw_type /  f'{transform_name}.py') if dw_type else transform_dir / f'{transform_name}.py'
    if function_path.exists():
        return str(function_path)
```

The Power of Paths Compels You!